.sync.yml

```diff
@@ -4,7 +4,7 @@
   docker_sets:
   - set: centos6-64
   - set: centos7-64
-  - set: debian8-64
+  - set: debian9-64
```

Comment:
to make the whole diff smaller, do you think it's possible to add debian9 in a seperate PR and drop debian 8 in a seperate PR?

Reply:
I could re-introduce those changes as a separate PR (unless someone else beats me to it) and rebase this one to clean things up, for sure

Reply:
Is a move to Debian 9 a requirement for Splunk 7.2? That would be the only reason to rebase. If it isn't a requirement then it should be able to be handled entirely separately.

Reply:
I only updated because the debian 8 docker images were defunct and not letting the test suite pass, I'm fine with ditching the change and ignoring the deb8 tests